src/wheel/bdist_wheel.py

```diff
@@ -178,6 +308,12 @@ def get_tag(self):
             if self.plat_name and not self.plat_name.startswith("macosx"):
                 plat_name = self.plat_name
             else:
```

Comment:
I don't see how the condition on line 308 can be satisfied, since `self.plat_name_supplied` is set to `self.plat_name is not None`. It would seem that `self.plat_name_supplied` can be removed?

Reply:
It was in code before #314.  But it somehow work, because I meet problem and need to create also #317. Maybe `plat_name` is set somehow in base class.